network-api/networkapi/wagtailpages/templates/wagtailpages/research_landing_page.html

```diff
@@ -1,64 +1,92 @@
+          <div class="tw-pt-5 tw-mr-4 medium:tw-mr-0 large:tw-w-full">
+            <h1>{{page.title}}</h1>
+            {% if page.intro %}
+              <p class="tw-body-large mt-3 mb-0">{{ page.intro }}</p>
+            {% endif %}
+            <div class="search-bar tw-my-5 tw-w-full">
+              <form action="library" method="get" accept-charset="utf-8" class="tw-w-full medium:tw-inline-flex">
+                <div class="tw-relative medium:tw-w-3/4">
+                  <label for="search" class="tw-sr-only">{% translate 'Search' context 'Button' %}</label>
+                  <input 
+                    type="text" 
+                    value="{{ search_query }}" 
+                    name="search" 
+                    id="search" 
+                    class="tw-w-full form-control tw-h-auto tw-p-4 tw-pr-7 tw-border-gray-40 tw-rounded-none" 
+                    placeholder="Search our research by keyword, author, or topic..." 
+                  />
+                  <button type="submit" class="tw-absolute -tw-translate-y-1/2 tw-top-1/2 tw-right-3 tw-text-blue-dark tw-bg-transparent">
+                    <svg fill="none" height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg" stroke="currentColor">
+                      <g stroke-linecap="round" stroke-linejoin="round" stroke-width="2">
+                        <circle cx="-14.5" cy="9" r="8" transform="scale(-1 1)" />
+                        <path d="m8.682 14.818-7.682 7.682" transform="matrix(-1 0 0 -1 9.682 37.318)" />
+                      </g>
+                    </svg>
+                  </button>
+                </div>
+              <a href="{% pageurl library_page %}" class="tw-block tw-self-center tw-font-bold tw-mt-3 medium:tw-px-5 medium:tw-mt-0">
+                Browse all
+              </a>
+            </div>
+          </div>
+
+          <div class="latest-research tw-pt-3 large:tw-pt-5">
+            <hr class="divider tw-border-black large:tw-mt-6 large:tw-hidden" />
+            <h2 class="large:tw-pb-2">{% trans "Latest Research" %}</h2>
+            <ul class="tw-list-none px-0 ">
+              {% for research_detail_page in latest_research_detail_pages %}
+                <li class="tw-py-4 small:tw-py-1">
+                  {% include "wagtailpages/fragments/research_detail_card.html" with research_detail_page=research_detail_page hide_related_topics=True landing_page=True%}
+                </li>
+              {% endfor %}
+            </ul>
+            <a href="{% pageurl library_page %}">
+              <button class="tw-btn-secondary tw-w-full medium:tw-w-auto large:tw-mt-4 large:tw-mb-7"> Browse all projects </button>
+            </a>
+          </div>
+        </div>
+  
+        <div class="browse-topics tw-col-span-12 medium:tw-px-4 large:tw-col-span-4">
+          <hr class="divider tw-border-black tw-mt-7" />
+          <h4 class="tw-h4-heading tw-pb-4">{% trans "Browse by topic" %}</h4>
```

Comment:
**Issue:** This may be styled like an H4, but from the semantic document structure this should be a H2. H4 would not only skip a level, but also mean it's a subheading of "latest research" (which is not the case).

Reply:
Updated!